tutorials/download_stablelm.md

```diff
@@ -39,3 +40,31 @@ pip install tokenizers
 
 python generate/base.py --prompt "Hello, my name is" --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b
 ```
+
+------
+
+### StableLM Zephyr 3B
+
+A new chat model representing the latest iteration in the series of lightweight LLMs, preference tuned for instruction following and Q&A-type tasks. This model is an extension of the pre-existing StableLM 3B-4e1t model and is inspired by the Zephyr 7B model from HuggingFace. With StableLM Zephyr's 3 billion parameters, this model efficiently caters to a wide range of text generation needs, from simple queries to complex instructional contexts on edge devices.
+More details can be found in the [anouncement](https://stability.ai/news/stablelm-zephyr-3b-stability-llm).
```

Comment:
anouncement -> announcement

Reply:
Oopsy, so embarrassing 😊 
VSCode tried to warn me, but all in vain.
<img width="933" alt="Screenshot 2023-12-09 at 7 15 20 PM" src="https://github.com/Lightning-AI/lit-gpt/assets/58434077/0790830a-17d3-40d3-a560-edda551239a2">

Good catch.
Thanks.